data/json/items/vehicle/motors.json

```diff
@@ -68,7 +68,7 @@
     "type": "GENERIC",
     "id": "motor_tiny",
     "name": { "str": "tiny electric motor" },
-    "description": "A tiny electric motor.  Useful for crafting.",
+    "description": "A small electric motor, suitable for powering a bicycle or a hedge trimmer.",
```

Comment:
https://www.amazon.com/Electric-Brushed-2750RPM-Scooter-Control/dp/B07KVVL1NM/ref=asc_df_B07KVVL1NM/?tag=hyprod-20&linkCode=df0&hvadid=317203115653&hvpos=&hvnetw=g&hvrand=14448846602955151091&hvpone=&hvptwo=&hvqmt=&hvdev=c&hvdvcmdl=&hvlocint=&hvlocphy=1026082&hvtargid=pla-635620784133&psc=1

The bike motor above is 4lbs so I believe mopeds and electric bicycles are probably more similar than they are different.

Reply:
That motor you linked is for one of them stand on e-scooters kids are using, which is more in line with a bicycle sized vehicle... 

When I think of a moped or scooter, I think of a vehicle which has the equivalent to a 50cc engine - like this one
![image](https://user-images.githubusercontent.com/34361592/230704388-bfa6b692-d646-4de0-94bc-c7d1c508c34f.png)

Perhaps I should change 'moped' on the bigger motor description to something thats more clearly higher tier, since the word can also mean a very small almost-bicycle in some places

Reply:
https://www.youtube.com/watch?v=5QYyzVyWckw  This guy shows 3 different motors which are the golf cart / 50cc bike equivalent.